public/js/CollisionManager.js

```diff
@@ -257,7 +260,7 @@ class CollisionManager {
 												 zerlinBox.x, zerlinBox.y, zerlinBox.width, zerlinBox.height);
 					if (zerlinCollision.collides) {
 						this.game.beams[i].isSizzling = true;
-						this.game.Zerlin.hits += this.game.clockTick * BEAM_HP_PER_SECOND;
+						this.game.Zerlin.hits += this.game.clockTick * dbConst.BEAM_HP_PER_SECOND;
```

Comment:
don't forget to subtract HP here, and make the status bar continuous instead of only supporting integers